frameworks/sveltekit/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "kit-prisma",
+  "version": "0.0.1",
+  "scripts": {
+    "dev": "svelte-kit dev",
+    "build": "svelte-kit build && cp node_modules/prisma/*query* .vercel_build_output/functions/node/render/;cp prisma/schema.prisma .vercel_build_output/functions/node/render/",
+    "package": "svelte-kit package",
+    "preview": "svelte-kit preview",
+    "prepare": "svelte-kit sync",
+    "check": "svelte-check --tsconfig ./tsconfig.json",
+    "check:watch": "svelte-check --tsconfig ./tsconfig.json --watch",
+    "vercel-build": "prisma generate && prisma migrate deploy && yarn build"
+  },
+  "devDependencies": {
+    "@sveltejs/adapter-auto": "next",
+    "@sveltejs/adapter-node": "^1.0.0-next.73",
+    "@sveltejs/adapter-vercel": "^1.0.0-next.47",
```

Comment:
Is this part of a default Sveltekit project?

Reply:
I believe so, they need this to be somewhat platform agnostic.